src/app/test_executive/README.md

```diff
@@ -1,120 +1,250 @@
-# Running Integration Tests
+# Gauntlet
 
-### Prerequisites Environment Setup
+## Overview
 
-Note: this environment setup assumes that one is a member of o(1) labs and has access to organization infrastructure.  You will need an o(1) labs GCP account and AWS account.
+*Gauntlet* is the name of Mina Protocol's fully end-to-end integration testing framework, developed in-house by O(1) Labs.  This piece of software is a standalone testing tool, and was previously known as simply "the integration testing framework" and sometimes as "the test executive".
+
+#### Elevator Pitch
+
+Gauntlet was created to tackle is the problem of testing a blockchain in as realistic of a way as possible.  Mina protocol, like all blockchains, require an entire decentralized, distributed network of peers.  Testing a blockchain, therefore, requires somehow creating a test network of nodes which interact with each other.  Creating a mocked peer with hardcoded responses is not sufficient for an integration test.  Simply spinning up a single node on one's local machine is a handy expedient for many development purposes but is also far from sufficient for testing purposes, because so much of a distributed computing network (such as a blockchain) consists in the interactions and consensus between nodes, and so having just one is not realistic.
+
+Given this, the solution, therefore, is quite clear: we must spin up from scratch an entire decentralized network of nodes, and then run tests against this network.  In concept this is quite simple, but of course in implementation it is ambitious.
+
+#### Structure (super high level)
+
+There are a number of moving parts to gauntlet, and it is also actively being developed and extended.  However in it's current state, Gauntlet tests Mina Protocol by creating a whole entire Mina testnet from scratch with a specified number of nodes of various types, creating a new blockchain from genesis, and then monitoring and interacting with this testnet in accordance to test logic as determined in pre-written tests; tests which are written in an Ocaml DSL which we have developed.  If the testnet seems healthy and behaves as one expects in response to the interactions, then the test passes, if not it fails.  Then the testnet is destroyed.  If one wants, one can prevent the destruction of the testnet with a flag, and then manually interact with the nodes on the testnet.
```

Comment:
Ocaml -> OCaml
```suggestion
There are a number of moving parts to gauntlet, and it is also actively being developed and extended.  However in it's current state, Gauntlet tests Mina Protocol by creating a whole entire Mina testnet from scratch with a specified number of nodes of various types, creating a new blockchain from genesis, and then monitoring and interacting with this testnet in accordance to test logic as determined in pre-written tests; tests which are written in an OCaml DSL which we have developed.  If the testnet seems healthy and behaves as one expects in response to the interactions, then the test passes, if not it fails.  Then the testnet is destroyed.  If one wants, one can prevent the destruction of the testnet with a flag, and then manually interact with the nodes on the testnet.
```

Reply:
thanks for the suggestions, have applied this change manually.